lib/lhm/entangler.rb

```diff
@@ -35,9 +35,9 @@ def entangle
 
     def untangle
       [
-        "drop trigger if exists `#{ trigger(:del) }`",
-        "drop trigger if exists `#{ trigger(:ins) }`",
-        "drop trigger if exists `#{ trigger(:upd) }`"
+        "drop trigger `#{ trigger(:del) }`",
```

Comment:
The purpose of using SQL statements with `IF EXISTS` is to prevent breaking replication in some edge cases.

Since this PR is already checking that the triggers exist before renaming the final table, the additional safety introduced here is already accomplished. 
Let's keep the `if exists` for these drop trigger statements.